code-samples/community/serving/helloworld-dart/README.md

```diff
@@ -17,67 +17,24 @@ that you can use for testing. It reads in the env variable `TARGET` and prints
 
 ## Recreating the sample code
 
-While you can clone all of the code from this directory, it is useful to know
-how to build a hello world Dart application step-by-step. This application can
-be created using the following instructions.
+While you can clone all of the code from this directory, we recommend you create
+your hello world Dart application by using the `dart` developer tool. This takes
+just a few steps:
 
-1. Create a new directory and write `pubspec.yaml` as follows:
+1. Create a new Dart app using the `server_shelf` template:
 
-   ```yaml
-   name: hello_world_dart
-   publish_to: none # let's not accidentally publish this to pub.dartlang.org
-
-   environment:
-     sdk: ">=2.12.0 <3.0.0"
-
-   dependencies:
-     shelf: ^1.0.0
+   ```shell
+   > create -t server-shelf helloworld-dart 
```

Comment:
Hi @mit-mit!
dart word seems to be missing in the command.

I came to this PR by traversing the following path - 

1. https://www.cloudskillsboost.google/course_sessions/1402704/labs/187125
   The hyperlink at the end of Task 2

2. https://cloud.google.com/run/docs/quickstarts/build-and-deploy
   Write the sample application -> Dart hyperlink

So, someone from the Cloud Team would need to update the above hyperlink once this goes live.

Reply:
> dart word seems to be missing in the command.

oh, good catch, fixed now